### DISCUSSION TITLE
[IntelliSense] Support for twMerge

### DISCUSSION THREAD

**Discussion by ThaJay** [tailwindlabs/tailwindcss#10851] [Ideas]
**What version of Tailwind CSS IntelliSense are you using?**

    Tailwind CSS IntelliSense: v0.9.9

**What version of Tailwind CSS are you using?**

    "tailwindcss": "~3.2.4"

**What editor are you using?**

    VsCode

**Describe your issue**

As shown in the screenshot example, tailwindcss-intellisense warns me about a cssConflict while the offending element className has no relation with the one used with twMerge.

![image](https://user-images.githubusercontent.com/9341747/227187591-85b7d606-9b9d-4500-b2b0-6834154d29f0.png)
![image](https://user-images.githubusercontent.com/9341747/227187657-7b75c03f-b3b7-4b8d-8265-779260d59cec.png)

**Comment by cannap** [comment]
hi check this  mayb it fixes
https://github.com/paolotiu/tailwind-intellisense-regex-list#tailwind-merge

**Reply by joaopedrodcf** [reply]
This is a life saver list of regex to make the intellisense work 💯

**Comment by ThaJay** [comment]
Thank you for the reply. It seems my cssConflict was already enabled for 'everywhere'. I don't see the config but that is how it seems to behave. If I add the linked regex to my config, it seems to get the same results where duplicate CSS properties are identified in unrelated strings.

**Comment by IskanderCodeerbedrijf** [comment]
@cannap Thanks! This worked for me
